src/components/PageContainer.tsx

```diff
@@ -13,7 +13,7 @@ interface PageContainerProps {
 
 function PageContainer({ children, className }: PageContainerProps) {
   return (
-    <div className={`container mx-auto space-y-10 px-4 pt-20 ${className}`}>
```

Comment:
this class is causing so much issues. The spacing can be handled individually at each page